crates/viewer/re_view_spatial/src/heuristics.rs

```diff
@@ -1,17 +1,57 @@
 use nohash_hasher::IntSet;
 use re_log_types::EntityPath;
 use re_types::ViewClassIdentifier;
-use re_viewer_context::ViewerContext;
+use re_viewer_context::{ViewClass as _, ViewerContext};
 
 use crate::view_kind::SpatialViewKind;
 use crate::visualizers::SpatialViewVisualizerData;
+use crate::{SpatialView2D, SpatialView3D};
 
-/// Returns all entities for which a visualizer of the given kind would be picked.
-///
-/// I.e. all entities for which at least one visualizer of the specified kind is "maybe visualizable"
-/// *and* has a relevant archetype.
-/// (we can't reason with "visualizable" because that can be influenced by view properties like its origin)
-pub fn default_visualized_entities_for_visualizer_kind(
+pub struct VisualizedEntities {
```

Comment:
would be nice to come up with a name that makes the heuristic nature of this more clear. Sounds too authorative otherwise